src/renderer/components/NewProjectModal/NewProjectView.tsx

```diff
@@ -97,7 +97,7 @@ const NewProjectView = ({ closeModal, nextView }: Props) => {
   return (
     <Container>
       <CustomColumnStack>
-        <CustomRowStack sx={{ alignItems: 'flex-start' }}>
+        <CustomRowStack paddingTop="5px" sx={{ alignItems: 'flex-start' }}>
```

Comment:
I think with mui its generally good practice to keep styling inside the sx prop object when possible.

paddingTop can be defined inside the of the sx object, as paddingTop or pt.